services/src/main/java/org/jboss/windup/web/services/model/MigrationProject.java

```diff
@@ -59,8 +58,17 @@
     @Column(name = "version")
     private int version;
 
-    @Column(length = 256, unique = true, nullable = false)
-    @Size(min = 1, max = 256)
+    /**
+     * The project is being created in the wizard and should not be treated as a ready project.
+     */
+    @Column(nullable = true)
+    @ColumnDefault("FALSE")
+    private boolean provisional = true;
+
+    @Column(length = 120, unique = false, nullable = false)
+    @Size(min = 3, max = 120)
+    @Pattern(message = "The name must be at least 3 letters long.",
+            regexp = "\\s*\\w[- \\w]+\\w\\s*") // Allow whitespace around, limit to A-Za-z0-9-, allow spaces inside, at least 3 characters after whitespace trimmed.
```

Comment:
https://github.com/windup/windup-web/commits/master/services/src/main/java/org/jboss/windup/web/services/model/MigrationProject.java - The last change to this before your PR was April fool's day. I think that you shouldn't use rebasing as an excuse in this case.

And why not fix it when it was pointed out? We are getting a lot of complaints about phrases used, so you know that things like that are heavily scrutinized. My suggestion for next time would be to initiate a conversation about how it could be changed to fit the requirements of the reviewer. That's just part of being a good team member, and I know that's what you want to be. :)